### ISSUE TITLE
"Member project of the .NET Foundation" at Level 3 - why?

### ISSUE THREAD

**Issue by shiftkey** [dotnet-foundation/project-maturity-model#10]
I've seen a lot of feedback and concerns about this mention, and I think it'd be great to be more explicit about why this is necessary or important at this step. I don't want to go straight to suggesting wording changes via a PR, so I figure I'd open this up to hear from others.

 - https://twitter.com/AzureTrenches/status/1176602875952283650

**Comment by shiftkey** [comment]
Having read over this section a few times, I guess the big reason why it's required at this stage is  from reading the contents of [`project-continuation-policies.md`](https://github.com/dotnet-foundation/project-maturity-model/blob/master/project-continuation-policies.md), so the Foundation can continue to support the project with maintenance updates if something happens to the original maintainer group.

Maybe making it clearer how these two are inter-twined would help?

**Comment by shiftkey** [comment]
I also found this comment over in [`maturity-ladder-policies.md`](https://github.com/dotnet-foundation/project-maturity-model/blob/master/maturity-ladder-policies.md#net-foundation-project-membership):

> Why link the ladder with Foundation membership? In short, this is how other foundations work, and it is considered a good model for growing projects from incubation to graduation for broad ecosystem benefit. However, the working group sees significant value in making the ladder a more generally applicable definition of quality for the ecosystem to use. That's why Foundation membership is part of the ladder but doesn't start with it. Level 3 was considered a good point to include Foundation membership as a required characteristic.

**Comment by richlander** [comment]
Yup. You largely answered your own question. A lot of thought went into this, both in terms of theory and practice.

The other foundations I looked at have maturity models and make them the maturity model for the foundation, so "foundation only". Foundations are either super general (Apache), scenario-specific (CNCF) or dev platform specific (.NET Foundation). For dev platform foundations, it makes more sense to look at that dev platform-related ecosystem more generally. So, that's why "foundation only" wasn't an attractive option for this. At the same time, it was attractive to position the maturity model as part of the .NET Foundation project membership criteria. Level 3 felt a good middle ground. Level 2 seemed too early and level 4 seemed too late.

You hit the nail on the head when you pointed to the continuation policies. Level 3 is where I wanted to see a big step function in consumer confidence. It's a lot easier to help a conservative organization adopt OSS if level 3 has a lot of checkboxes checked. .NET Foundation membership checks two main boxes: IP ownership is fully understood and known; and the project is aligned with the values and goals of the .NET Foundation.

In summary, making this both an .NET ecosystem thing and a .NET Foundation membership thing is inherent to the proposal. Making foundation membership start at level 3 is arbitrary (although based on a rationale, as described above). Having only four levels is intentional (not inherent) to make it simple. Starting foundation at level 4 remains an option and feedback will help with decisions like that.

Does that help?

**Comment by shiftkey** [comment]
Can we make it clearer in the preamble (whatever level this ends up at) about this context/motivation? I think expanding the doc to help readers better understand the transitions, without having to pore over multiple docs, would help here...

**Comment by richlander** [comment]
Good point.
